src/message.js

```diff
@@ -186,14 +184,16 @@ Globalize.prototype.messageFormatter = function(pathOrMessage) {
     }
 
     arguments[0] = sanitizedPathExists ? sanitizedPath : pathOrMessage;
-    return messageFormatterSuper.apply(this, arguments);
+    return globalizeMessageFormatter.apply(this, arguments);
 };
 
-export default React.createClass(generator("formatMessage", ["path", "variables"], {
+export default generator("formatMessage", ["path", "variables"], {
     beforeFormat: function() {
-        messageSetup(this.props, this.instance, this.args);
+        if (process.env.NODE_ENV !== "production") {
+            messageSetup(this.globalize, this.props, this.globalizePropValues);
```

Comment:
I'm not sure all of this could be considered development only. I believe we could for only part of that funciton (see an `if` block with `Development mode only` comment on top). I believe our common use case of using default messages will work fine, but `path` and `scope` won't work.

On test/formatMessage.spec.js we have tests for `path`, can we add one using `NODE_ENV=production` env?